src/Kunstmaan/MultiDomainBundle/Tests/Helper/DomainConfigurationTest.php

```diff
@@ -318,6 +317,24 @@ private function getEntityManager()
         return $em;
     }
 
+    private function getAdminRouteHelper()
```

Comment:
There is no need for testing the isAdminRoute method from the AdminRouteHelper again in the DomainConfigurationTest in my opinion. We moved the logic into a service for this exact reason, I tested the AdminRouteHelper on it's own so we can assume that service works as intended. Therefore we can mock it here and not have to worry about anything else than the functionality of the DomainConfiguration.